client/src/App.js

```diff
@@ -3,31 +3,143 @@ import { Switch, Route } from 'react-router-dom';
 import { saveToLocalStorage, loadFromLocalStorage } from './lib/localStorage';
 import styled from 'styled-components/macro';
 import Home from './pages/Home.js';
-import BurgerMenu from './BurgerMenu.js';
+import BurgerMenu from './components/BurgerMenu.js';
 import MarketCard from './pages/MarktCard.js';
 import MarketForm from './pages/MarktForm.js';
```

Comment:
It is often seen as good practice to structure the imports from outside in:
First external imports (e.g. `styled`)
And then your own components, libs etc. I would also recommend that you group them (all pages together) and sort them alphabetically. Just gives a very structured impression :)

Reply:
Thnak you Miriam i changed it now